src/components/Header/Header.test.tsx

```diff
@@ -128,3 +145,124 @@ describe("Header", () => {
     expect(header).toMatchSnapshot();
   });
 });
+
+describe("Patron is logged in", () => {
```

Comment:
Set a variable to store the `Cookies.get` function before `beforeAll`:
`const realGet = Cookies.get;`

Instead of spying on the "get" function, mock it:
`Cookies.get = jest.fn().mockReturnValue(mockCookieValue);`

Restore the original `Cookies.get` function:
```
afterAll(() => {
    Cookies.get = realGet;
  });
```

and finally test:
`expect(Cookies.get).toHaveBeenCalledWith("nyplIdentityPatron");


I think it's enough if this is done once instead of all three tests but that's up to you.